test/fixture/long-stack-trace/test.js

```diff
@@ -0,0 +1,14 @@
+import test from '../../../';
+const Promise = require('./enable-trace.js');
+
+test(async t => {
+	const resolve = await Promise.resolve().then(function outer() {
+    return Promise.resolve().then(function inner() {
```

Comment:
Indentation seems a bit off here, though looks like we're no linting this file so hey 🤷‍♂️